moto/core/exceptions.py

```diff
@@ -63,8 +64,7 @@ def __init__(
         self.message = message
 
         if template in self.templates.keys():
-            env = Environment(loader=DictLoader(self.templates))
-            self.description: str = env.get_template(template).render(
+            self.description: str = self.env.get_template(template).render(
```

Comment:
```suggestion
            self.description: str = self.__class__.env.get_template(template).render(
```
Functionally equivalent (in this case), but this would make it more obvious to a developer that we're calling a class-attribute.